lua/entities/gmod_wire_expression2/init.lua

```diff
@@ -131,16 +131,16 @@ function ENT:Execute()
 
 	local bench = SysTime()
 
-	local ok, msg = pcall(selfTbl.script, context)
+	local ok, msg = pcall(script or selfTbl.script, args or context)
```

Comment:
Should above be `local context = args or selfTbl.context`? Lots of other places in this function using selfTbl.context even if args is provided.

Reply:
Ig that's made to target context of entity nonetheless the context passed? Also that's how Vurv's implementation looks like. He uses args and script only in the pcall. Altho maybe I've missed something?

Reply:
@Vurv78 can u check

Reply:
Those changes were done post my pr.

Reply:
@Vurv78 No idea what you mean or if that's a good thing or not. Can you add your review?

Reply:
As in, my changes were done before the selfTbl changes, and he merged my changes without properly addressing the merge conflicts, so it should use selfTbl.

It doesn't really do anything special with self or selfTbl, so I'd recommend just reimplementing my change which is adding an optional function parameter to the Execute method that defaults to the compiled function in self

Reply:
So you're saying the context var should be the input arg but default to selfTbl.context if no input arg. Then that change is also needed.

Reply:
Done